aries-site/src/data/structure.js

```diff
@@ -138,7 +138,7 @@ export const structure = [
   },
   {
     name: 'Extend',
-    color: 'black',
+    color: 'text',
```

Comment:
```suggestion
    color: 'text-strong',
```

Reply:
I'm good with that. The design calls for #000 in light mode, which is text-strong. So it seems fitting to use text-strong.